portfolio/src/main/webapp/mainPage.html

```diff
@@ -0,0 +1,205 @@
+
+
+    </div>
+
+    <div class="w3-container">
+        <hr>
+    </div>
+</div>
+<button onclick="myAccFunc('showExperience')" class="w3-padding-16 w3-theme w3-button w3-block w3-center" style="font-size:x-large;">Experience</button>
+<div id="showExperience" class="w3-hide">
+
+    <div class="w3-row-padding w3-center w3-margin-top">
+        <div class="w3-third">
+            <div class="w3-card w3-container" style="min-height:460px">
+                <img src="/images/santaFe.jpg" style="width:100%" />
+                <h3>Santa Fe College | Gainesville, FL</h3>
+                <p>Summer 2019 – Summer 2020 <br>
+                    Math Tutor<br>
+                    •College Reading & Learning Association (CRLA) Level 1 certified
+                    tutor <br>
+                    •Collaborated with students from various math courses<br>
+                    •Assisted students to create studying strategies while supporting them with classwork
+                </p>
+            </div>
+        </div>
+
+        <div class="w3-third">
+            <div class="w3-card w3-container" style="min-height:460px">
+                <img src="/images/sitel.jpg" style=" width:100% ;height:100%" />
+                <h3>SITEL | Bogota, Colombia</h3>
+                <p> [13 weeks] 2019 <br>
+                    Bilingual Call Center Agent<br>
+                    • Agent as a representative of Just Energy Group Inc.<br>
+                    • Evaluated customer’s info to explore solutions while maintaining high-quality service<br>
+                    • Inbound calls sales team: Promoted products and services to customers during calls<br>
+                </p>
+            </div>
+        </div>
+
+    </div>
+</div>
+<button onclick="myAccFunc('showAboutMe')" class="w3-padding-16 w3-theme w3-button w3-block w3-center" style="font-size:x-large;">About</button>
+<div id="showAboutMe" class="w3-hide">
+    <div class="w3-third">
+        <div class="w3-card w3-container">
+            <img src="/images/Pena, Thomas.jpg" style="width:100%" />
+        </div>
+    </div>
+    <div class="w3-third">
+        <div class="w3-card w3-container">
+            <h2 class="w3-left-align">Who am I?</h2>
+            Hi! I'm Thomas. I'm from Bogota, Colombia
+            and I'm currently studying at the University of Florida.<br />
+            Technology and innovation has always fascinated me that is why I'm
+            majoring in Computer Science!<br />
+            Some fun facts about me: <br />• Oreo lover <br />
+            • I speak three languages<br />
+            • Currently training to run a marathon by the end of this year!
+        </div>
+    </div>
+</div>
+<button onclick="myAccFunc('showContact')" class="w3-padding-16 w3-theme w3-button w3-block w3-center" style="font-size:x-large;">Contact me</button>
+<div id="showContact" class="w3-hide w3-black">
+    <div class="container">
+        <div style="background-color:#27323a">
+
+
```

Comment:
Looks like you have some extra spaces here, if you need new lines please use <br> instead.